_posts/2016/02/2016-02-03-swdcretreat2015-university_course.html

```diff
@@ -0,0 +1,437 @@
+  It enables them to make a plot which gets them excited.
+  <em>Then</em> you can make a web page.
+  <em>Then</em> you can easily put it on Github.
+  It makes it extremely rewarding to the students.
+  The peer review is helpful because they can provide feedback and have constant interaction between people.
+  The students discuss the layout of the repository, how easy was it to find scripts and understand what the code is doing, etc.
+</p>
+
+
+<h4>Grading code and technical work is difficult, how do you do it?</h4>
+
+<p>
+[J] One of other reasons of doing peer review is to get feedback to the students faster than if only the instructor is grading.
+</p>
+
+<h4>Jenny since you seem to have been doing this for a few years, do you have any available resources to share?</h4>
+
+<p>
+  [J] My writing on the teaching of the course has always been talks.
+  My Courses are online, and I have speaker decks.
+  <ul>
+    <li><a href='http://stat545-ubc.github.io/'>Data wrangling, exploration, and analysis with R</a></li>
+    <li><a href='http://stat540-ubc.github.io/'>Statistical Methods for High Dimensional Biology</a></li>
+  </ul>
+  There's also Ethan White's courses <a href='http://www.datacarpentry.org/semester-biology/syllabus/'>here</a> and <a href='http://www.programmingforbiologists.org/'>here</a>.
+<p>
+
+
+<h4>Software and Stats for biology is such a fast paced field, how do you even decide what to teach?</h4>
+
+<p>
+  [J] The 2 courses I've taught the past decade is "Data Analysis" where I am chasing technology.
+  It's exhausting, but I also want to learn it myself.
+  <br>
+  I currently do the opposite in my genomics class.
+  I started by chasing the technology for years, but every year almost required a completely new course re-write.
+  I eventually generalized it to the key genomics pipelines and statistical analysis.
+  For example, multiple comparison and empirical bayes.
+  I try not to chase specific sequencing platforms.
+  You can only be chasing technology a few places at a time.
+  <br>
+  Research wise, I've only been able to put these classes after I've gotten tenure.
+  This allmws me to put more effort into developing the course with fewer outside distractions.
+  I would say what I'm doing would not mesh well with being super productive research-wise
+  ***Jeff Leek***, Roger Peng, rafa irizarry, are doing on line courses and super productive research-wise.
+</p>
+
+
+<h4>Do you have TAs</h4>
+
+<p>
+  [J] An <em>army</em> of TAs is critical.  I have 6-7 (not all full time).
+  The department here was able to prove that this is not just a statistics class anymore, and people are coming from all over campus.
+  So I got external funding for more TAs than the average graduate class.
+  I only have graduate TAs, no undergraduate TAs.
+</p>
+
+<p>
+  [T] Unfortunately Quest is a very new institute with limited funding.  So I am the only TA/Helper/Teacher/Office Hours.
```

Comment:
I misspoke here, please change this response to: "During class I have no helpers or TAs, but outside of the class there are are peer helpers available."